src/css/_mixins.scss

```diff
@@ -1,6 +1,4 @@
-
-// @mixin make-font($size, $weight, $color) {
-//   font-size: $size;
-//   font-weight: $weight;
-//   color: $color;
-// }
+@mixin add-hover-state($cursor-state) {
```

Comment:
Great mixin, I haven't even thought of the hover states yet. I didn't quite see a commit for sass styling, you've made a lot of changes and your commits mostly are about your class files. I think making commits about these changes is important too.

Reply:
Yes, I made all of those commits before we were paired up as project buddies. I will definitely make commits for any additional stylings changes :)